src/transformers/pipelines/depth_estimation.py

```diff
@@ -87,7 +87,7 @@ def _sanitize_parameters(self, **kwargs):
     def preprocess(self, image):
         image = load_image(image)
         self.image_size = image.size
-        model_inputs = self.feature_extractor(images=image, return_tensors=self.framework)
+        model_inputs = self.image_processor(images=image, return_tensors=self.framework)
```

Comment:
backward compatibility is ensured by the change (of `__init__`) in `base.py`, see comment there.